README.md

```diff
@@ -82,7 +82,7 @@ user@host:~$ sudo python3 packet_sniffer.py
 
 - Sample output captured during execution:
 
-```
+```sh
```

Comment:
Can you remove this marker? It is highlighting parts of this code block as if they were shell scripts, which they are not.

Reply:
Made a new commit with the required changes